app/assets/javascripts/account/AccountService.js.coffee

```diff
@@ -308,6 +319,11 @@ AccountService = (
   Service.afterUserTokenValidationTimeout = ->
     Service.accountSuccess.messages.userTokenValidationTimeout = $translate.instant('SIGN_IN.USER_TOKEN_VALIDATION_TIMEOUT')
 
+  Service.emailAlreadyInUse = (email = '') ->
```

Comment:
@callaghanc I think this function was leftover as a placeholder? can remove if so.

Reply:
@kaptron yep that's a leftover, good catch, will take it out